inst/css/gt_styles_default.scss

```diff
@@ -366,6 +366,14 @@
     font-size: 65%;
   }
 
+  .gt_two_val_uncert {
+    display: inline-block;
+    line-height: 1em;
+    text-align: right;
+    font-size: 60%;
+    vertical-align: -0.25em;
```

Comment:
Add left margin to this class, and dispose of the outer span

Reply:
Now done.